packages/system/src/Item/utils.ts

```diff
@@ -50,7 +50,5 @@ export const toItem = (line: ItemLike): Item => ({
     averageMonthlyConsumption: 0,
     availableStockOnHand: 0,
   },
-  unitName:
-    ('lines' in line ? line.lines[0].item?.unitName : line.item?.unitName) ??
-    '',
+  unitName: '',
```

Comment:
did you have thoughts about how to bring the unit back?
The inconsistency of `itemName` `itemCode` etc and the `item` as an object  bothers me, and is made slightly more annoying because `itemUnitName` is missing. Wonder if we have to map the item on to the ItemLike object?

Reply:
Ah nice, thanks. I should have added a TODO to come back before I PRd - I commented this out to make the typings easier to handle at one point and forgot 😁 

It's not too bad to fix now. Previously we omitted the `item` field on `InvoiceLine` because it was so annoying (having all the required fields like `availableBatches` etc) - now `OutboundLine` and `InboundLine` have an `item` field so we can go back to just doing `lines[0].item.unitName : line.item.unitName`

(I'll do this is in the next PR 😁 )